cmd/samples/recipes/ctxpropagation/propagator.go

```diff
@@ -90,3 +91,12 @@ func (s *propagator) ExtractToWorkflow(ctx workflow.Context, reader workflow.Hea
 	}
 	return ctx, nil
 }
+
+// SetValusTestHeader places the Values container inside the header 
+func SetValuesTestHeader(values Values, header *shared.Header) {
```

Comment:
nit: probably remove `Test` from the name if it's defined here? or maybe just move to the test file.

Also if the purpose of exporting this func is for reusing it somewhere else, it might be a good idea to return an error when encoding failed or header is nil.

Reply:
done